cli/src/types.rs

```diff
@@ -38,7 +38,11 @@ impl FromStr for Role {
 
 #[derive(Clone, Debug, Deserialize)]
 pub struct GithubRelease {
-    pub name: String,
+    // The `name` entry stores the GitHub Release name,
+    // which could be set to something other than the version.
+    // Using the `tag_name` entry is better since the tags
+    // are much more tightly coupled with the release version.
```

Comment:
This is great context to have in the PR or in a commit message, but I don't think this needs to live as a comment in the code.

```suggestion
```